docs/reference/features/compliant.md

```diff
@@ -0,0 +1,54 @@
+# Compliant Cloud
+
+
+> :material-check: Feature is in production and fully supported
+>
+> :material-timer-sand: Feature is in deployment, not yet supported
+>
+> :material-logout: Feature is deprecated, being phased out
+>
+> :material-close: Feature is not available
+
+
+## Virtualization
+
+|               | Sto1HS                | Sto2HS                |
+| ------------- | ----------------      | --------------------- |
+| Dedicated CPU | :material-close:      | :material-close:      |
+| Virtual GPU   | :material-timer-sand: | :material-timer-sand: |
+
+
+## Block storage
+
+|                                | Sto1HS           | Sto2HS           |
+| ------------------------------ | ---------------- | ---------------- |
+| Highly available storage       | :material-check: | :material-check: |
+| High-performance local storage | :material-close: | :material-close: |
+| Boot from persistent storage   | :material-check: | :material-check: |
```

Comment:
Um, what is that and how different from HA storage? Like ephemeral vs volumes?

Reply:
"Highly available storage" is ephemeral storage that survives a compute node outage.

"Boot from persistent storage" is boot-from-volume.

Reply:
Maybe it's just my perception, but it sounds like different things. So I'd understood that persistent storage is not HA and HA is not persistent.
Also saying about boot-from-volume separately makes quite limited sense to me as I actually don't see the way to prohibit it. Well, ofc except overriding policies:)

Reply:
Ok so should I just remove the "boot from persistent storage" row?

Reply:
Removed in 9a78b693cb91f00d0d5fc35d6fd6fbc75ad29f14.